csharp/ql/test/query-tests/API Abuse/NoDisposeCallOnLocalIDisposable/NoDisposeCallOnLocalIDisposable.cs

```diff
@@ -70,14 +70,6 @@ public IDisposable Method()
         using (var reader = new StreamReader(new FileStream("", FileMode.Open)))
             ;
 
-        // GOOD: XmlDocument.Load disposes incoming XmlReader (according to CIL)
-        var xmlReader = XmlReader.Create(new StringReader("xml"), null);
-        var xmlDoc = new XmlDocument();
-        xmlDoc.Load(xmlReader);
-
-        // GOOD: Passed to a library. This is only detected in CIL.
-        DisposalTests.Class1.Dispose(new StreamWriter("output.txt"));
-
```

Comment:
Why are these test cases removed? We could mark them as false negatives, couldn't we?

Reply:
I will re-add the tests.